frontend/src/component/admin/banners/BannersTable/BannerDeleteDialog.tsx

```diff
@@ -0,0 +1,31 @@
+import { Dialogue } from 'component/common/Dialogue/Dialogue';
+import { IInternalBanner } from 'interfaces/banner';
+
+interface IBannerDeleteDialogProps {
+    banner?: IInternalBanner;
+    open: boolean;
+    setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    onConfirm: (banner: IInternalBanner) => void;
+}
+
+export const BannerDeleteDialog = ({
+    banner,
+    open,
+    setOpen,
+    onConfirm,
+}: IBannerDeleteDialogProps) => (
+    <Dialogue
+        title='Delete banner?'
+        open={open}
+        primaryButtonText='Delete banner'
+        secondaryButtonText='Cancel'
+        onClick={() => onConfirm(banner!)}
+        onClose={() => {
+            setOpen(false);
+        }}
+    >
+        <p>
+            You are about to delete banner: <strong>{banner?.message}</strong>
```

Comment:
Nit: if banner's not present, then this message will be odd

Reply:
True, however this is consistent with other similar dialogs we have. 
Maybe we should change the logic a bit, assume `banner` is always present in the dialog and leave it to the parent to make sure of that, or in this case, make TS happy.